packages/driver/src/cy/commands/actions/scroll.coffee

```diff
@@ -221,15 +221,34 @@ module.exports = (Commands, Cypress, cy, state, config) ->
         $utils.throwErrByPath("scrollTo.invalid_target", {args: { x, y }})
 
       if options.log
-        deltaOptions = $utils.filterOutOptions(options, {duration: 0, easing: 'swing'})
+        deltaOptions = $utils.stringify(
```

Comment:
If I don't stringify the options, the concatenated result appears like `0, 200, Object{3}`, since internal to the logging module we are using the original args to the function.